### PR TITLE
Creating a namespace for the dev env of the check-my-diary app

### PR DESCRIPTION
**Pull request by ewastempel**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+kind: Namespace
+metadata:
+  name: check-my-diary-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "dev"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/application: "check-my-diary"
+    cloud-platform.justice.gov.uk/owner: "Ewa Stempel: ewa.stempel@digital.justice.gov.uk"
```
[the remark this answers is not in the document]
Comment (ewastempel) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/00-namespace.yaml]: OK, I am not sure who will be the end user/owner yet, but how about I set it to our team for the moment?
`Digital Studio Operations Team : digital-studio-operations-team@digital.justice.gov.uk` ? Would that work for you?

[the remark this answers is not in the document]
Comment (sgsollie) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/00-namespace.yaml]: @ewastempel owner will be checkmydiary@digital.justice.gov.uk

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/01-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: check-my-diary-dev-admin
+  namespace: check-my-diary-dev
+subjects:
+  - kind: Group
+    name: "github:check-mny-diary"
```
Comment (sgsollie) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/01-rbac.yaml]: watch the typo :)

also add admin permissions for dso in sheffield

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/00-namespace.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ewastempel) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/00-namespace.yaml]: I've set it to check-my-diary-team@digital.justice.gov.uk, as discussed

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/01-rbac.yaml (hunk 2 above)
Comment (ewastempel) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/01-rbac.yaml]: fixed the typo and added DSO to admins

**Pull request merged**